common/src/api/external/mod.rs

```diff
@@ -2459,6 +2459,10 @@ pub struct SwitchPortRouteConfig {
 
     /// The route's gateway address.
     pub gw: IpNet,
+
+    /// The VLAN identifier for the route. Use this if the gateway is reachable
+    /// over an 802.1Q tagged L2 segment.
+    pub vid: Option<u16>,
```

Comment:
_tiny_ nit, `vlan_id` would probably be more recognizable at a glance.

Reply:
done